package.json

```diff
@@ -23,7 +23,7 @@
       "ccdUrl": "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
     },
     "preview": {
-      "cosUrl": "http://finrem-cos-pr-567.service.core-compute-preview.internal",
+      "cosUrl": "http://finrem-cos-pr-584.service.core-compute-preview.internal",
```

Comment:
Reminder to revert before merge!

Reply:
k, is there a reason why it's pointing to `567`?

Reply:
https://github.com/hmcts/finrem-case-orchestration-service/pull/567
Its supposed to be kept up to date with latest COS Master - master branch for functional tests in CCD repo. 
I believe its necessary for respondent solicitor journey